src/core/mac/mac.hpp

```diff
@@ -820,6 +818,10 @@ class Mac : public InstanceLocator, private NonCopyable
 #if OPENTHREAD_CONFIG_MAC_CSL_TRANSMITTER_ENABLE
     TimeMilli mCslTxFireTime;
 #endif
+#endif
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
+    uint8_t  mCslChannel;
```

Comment:
Add a comment: `// When Mac::mCslChannel is 0, it indicates that CSL channel hasn't been specified by the upper layer.`